cgi/user.pl

```diff
@@ -236,12 +236,37 @@
 				push @{$team_section_ref->{fields}}, {
 					field => "team_". $i,
 					label => sprintf(lang("team_s"), $i),
-				 };
+				};
 			};
 
 			push @{$template_data_ref->{sections}}, {%$team_section_ref};
 		}
 
+		# Contributor section (not in pro)
```

Comment:
Just curious: what's the reason for not showing the options in the producers platform?